__tests__/main.test.ts

```diff
@@ -17,22 +17,54 @@ function wrapInClassInMainMethod(
 `;
 }
 
-/*
-import * as process from 'process'
-import * as cp from 'child_process'
-import * as path from 'path'
+jest.setTimeout(20 * 1000);
```

Comment:
A better alternative would be to set timeout for individual tests like given [here](https://github.com/facebook/jest/issues/5055#issuecomment-350827560).

Reply:
Good find, let's do that!

Reply:
If there are many tests for which would need to do this fix, you can create a separate PR for it. Up to you.

Reply:
Actually, since all of the main tests require a high timeout, it's probably just a better idea to set the timeout globally.

Reply:
In that case, you can configure this in [jest.config.js](https://github.com/SpoonLabs/sorald-buildbreaker/blob/main/jest.config.js). Haven't looked up if it is possible but if it is, I think you should consider it.